native/dggsvp3.go

```diff
@@ -260,7 +260,7 @@ func (impl Implementation) Dggsvp3(jobU, jobV, jobQ lapack.GSVDJob, m, p, n int,
 		}
 
 		// Clean up A.
-		for i := k + 1; i < k+l; i++ {
+		for i := k + 1; i < min(m, k+l); i++ {
```

Comment:
I'm still not convinced that this is completely correct. If k+l<m, then this loop will not zero out the bottom right block in A, unlike in the reference.